src/libraries/Microsoft.PowerFx.Interpreter/Marshal/TypeCoercionProvider.cs

```diff
@@ -48,6 +52,18 @@ public static bool TryCoerceTo(this FormulaValue value, out StringValue result)
             return TryCoerceTo(value, CreateFormattingInfo(), out result);
         }
 
+        /// <summary>
+        /// Try to convert value to String format.
+        /// </summary>
+        /// <param name="value">Input value.</param>
+        /// <param name="runtimeConfig">Runtime Config. Need to SetCulture, SetCancellationToken and SetTimeZone for RuntimeConfig.</param>
+        /// <param name="result">Result value.</param>
+        /// <returns>True/False based on whether function can convert from original type to String type.</returns> 
+        public static bool TryCoerceTo(this FormulaValue value, RuntimeConfig runtimeConfig, out StringValue result)
+        {
+            return TryText(CreateFormattingInfoFromRuntimeConfig(runtimeConfig), IRContext.NotInSource(FormulaType.String), value, null, out result);
```

Comment:
>TryText [](http://example.com/codeflow?start=19&length=7)

nit - can this "chain to" 
TryCoerceTo(this FormulaValue value, FormattingInfo formatInfo, out StringValue result) ?

Reply:
Yes, updated the code.